clients/java/src/main/java/io/camunda/client/impl/worker/JobStreamerImpl.java

```diff
@@ -198,6 +253,19 @@ private void lockedHandleStreamComplete(final Throwable error) {
     }
 
     if (error != null) {
+      if (error instanceof StatusRuntimeException) {
+        final StatusRuntimeException statusError = ((StatusRuntimeException) error);
```

Comment:
🙃 I don't think the wrapping parens are required here, but if they are, TIL.


```suggestion
        final StatusRuntimeException statusError = (StatusRuntimeException) error;
```